app/src/main/res/layout/activity_checkout.xml

```diff
@@ -36,18 +36,37 @@
                 android:textAppearance="?textAppearanceHeadline5"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toTopOf="parent"
+                android:visibility="invisible"
+                tools:visibility="visible"
                 tools:text="Travel Insurance" />
 
             <TextView
-                android:id="@+id/subtitle"
-                android:layout_width="match_parent"
+                android:id="@+id/original_cost"
+                android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:layout_marginHorizontal="@dimen/base_margin_double"
+                android:layout_marginStart="@dimen/base_margin_double"
                 android:layout_marginTop="@dimen/base_margin"
                 android:textAppearance="?textAppearanceHeadline6"
-                android:textColor="?android:textColorPrimary"
+                android:textColor="?android:textColorHint"
+                android:visibility="invisible"
```

Comment:
Should this one be `gone`?

Reply:
Getting a nicer animation with TransitionManager (alpha instead of translation) if this is invisible :)

Reply:
Sensible!